src/WCSimPMTObject.cc

```diff
@@ -20,7 +20,46 @@
 #include <cstring>
 #include <iostream>
 
-PMT20inch::PMT20inch(){}
+
+////////////////////////////////////////////////////////////////////////////////////////////////
+// PMT Base Class
+
+WCSimPMTObject::WCSimPMTObject()
+{
+    collectionEfficiencyAngle = { 0., 10., 20., 30., 40., 50., 60., 70., 80., 90.};
+    collectionEfficiency      = { 100., 100., 100., 100., 100., 100., 100., 100., 100., 100.};
+}
+
```

Comment:
Previously there was no constructor in the base class.  Now there is a constructor which is called first, setting the collection efficiency to 100%, which can then be overridden in the derived class constructor.  I've added a comment which explains this.